src/app/modules/training-event/services/event.service.spec.ts

```diff
@@ -1,22 +1,92 @@
 import { TestBed } from '@angular/core/testing';
 import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { Subject } from 'rxjs';
 
 import { EventService } from './event.service';
 import { environment } from 'src/environments/environment';
 import {
+  CampusEventResponse,
   OffCampusEventRequest,
   OffCampusEventResponse,
 } from 'src/app/interfaces/event';
+import { AUTH_SERVICE } from 'src/app/interfaces/auth-service';
+import { PaginatedResponse } from 'src/app/interfaces/paginated-response';
 
 describe('EventService', () => {
   let httpTestingController: HttpTestingController;
+  const dummyEvent: CampusEventResponse = {
```

Comment:
You don't need real data in these tests, consider replace them with `req.flush({count: 2});`